pkg/storaged/stratis-details.jsx

```diff
@@ -468,6 +468,8 @@ export const StratisPoolDetails = ({ client, pool }) => {
     const sidebar = <StratisPoolSidebar client={client} pool={pool} />;
 
     function render_fsys(fsys, offset, total) {
+        const overhead = pool.TotalPhysicalUsed[0] ? (Number(pool.TotalPhysicalUsed[1]) - total) : 0;
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18831-20230522-125636-1ec6e298-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)